Examples/Tests/Langmuir/inputs_2d_multi_rz_rt

```diff
@@ -13,9 +13,10 @@ amr.max_level = 0
 
 # Geometry
 geometry.coord_sys   = 1                  # 0: Cartesian
-geometry.is_periodic = 0     1            # Is periodic?
 geometry.prob_lo     =   0.e-6   -20.e-6    # physical domain
 geometry.prob_hi     =  20.e-6    20.e-6
+boundary.field_lo = none periodic
+boundary.field_hi = none periodic
```

Comment:
Here as well, changed the input to use the new interface with `none` for rmin and rmax